webextension/js/background/messageHandler.js

```diff
@@ -120,6 +128,29 @@ const messageHandler = {
     });
   },
 
+  async incrementCountOfContainerTabsOpened() {
+    const key = "containerTabsOpened";
+    const count = await browser.storage.local.get(key);
```

Comment:
I think you can use `get({[key]: 1})` right?

Reply:
Instead of the presence checking below etc

Reply:
updated.